task/sast-coverity-check-oci-ta/0.3/MIGRATION.md

```diff
@@ -0,0 +1,8 @@
+# Migration from 0.2 to 0.3
+
+- The `IMAGE_DIGEST` parameter has been added.
+
+## Action from users
+
+- NA
```

Comment:
Could it break user's pipeline if they were passing full image reference in `IMAGE_URL`?

Reply:
same here